ui/component/preorderButton/view.jsx

```diff
@@ -65,12 +80,86 @@ export default function PreorderButton(props: Props) {
 
   // populate customer payment data
   React.useEffect(() => {
-    checkIfAlreadyPurchased();
-  }, []);
+    console.log('starting the check now');
+    checkIfAlreadyPurchasedOrPreordered();
+  }, [claimId]);
+
+  let preorderOrPurchase;
+  let pastTense;
+  if(purchaseTag){
+    preorderOrPurchase = 'purchase'
+    pastTense = 'purchasing'
+  } else {
+    preorderOrPurchase = 'preorder'
+    pastTense = 'preordering'
+  }
 
   return (
     <>
-      {preorderTag && !hasAlreadyPreordered && !myUpload && (
+      {/* purchasable content, not preordered and still needs to be purchased */}
+      {purchaseTag && !hasAlreadyPreorderedOrPurchased && !myUpload && !preorderContentClaim && (
+        <div>
+          <Button
+            iconColor="red"
+            className={'preorder-button'}
+            icon={fiatIconToUse}
+            button="primary"
+            label={__('This content can be purchased for $' + purchaseTag, {
+              fiatSymbolToUse,
+              preorderTag,
+            })}
+            requiresAuth
+            onClick={() => doOpenModal(MODALS.PREORDER_CONTENT, {
+              uri,
+              checkIfAlreadyPurchased: checkIfAlreadyPurchasedOrPreordered,
+              preorderOrPurchase,
+              purchaseTag
+            })}
+          />
+        </div>
+      )}
+      {/* purchasable content, already purchased or preordered */}
+      {purchaseTag && hasAlreadyPreorderedOrPurchased && !myUpload && !preorderContentClaim && (
+        <div>
+          <Button
+            iconColor="red"
+            className={'preorder-button'}
+            icon={fiatIconToUse}
+            button="primary"
+            label={__(`Thanks for ${pastTense}, enjoy your content! (No expiry date)`, {
+              fiatSymbolToUse,
+              preorderTag,
+            })}
+            requiresAuth
+          />
```

Comment:
Saw this from the GUI:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/64950861/181497665-2601ca08-2a0e-4159-96c8-e0462ca6e5d9.png">

- The primary colors are for buttons, so this violates some design rules.
- Then realized it's actually a button (cursor changes), but it doesn't do anything.  If there's no action, it should just be a span.  Can use things like `help--notice` or other similar things to make it a banner.  Rave should have the answer on what's best.
- You mentioned about fixing strings, so just a reminder that this is violating the no-template-literal rule.